client/landing/stepper/index.tsx

```diff
@@ -58,6 +59,7 @@ function initializeCalypsoUserStore( reduxStore: any, user: CurrentUser ) {
 	config.isEnabled( 'signup/inline-help' ) && reduxStore.dispatch( requestHappychatEligibility() );
 	reduxStore.dispatch( setCurrentUser( user ) );
 	reduxStore.dispatch( requestSites() );
+	reduxStore.dispatch( setCurrentFlowName( 'link-in-bio' ) ); //make it dynamic ofc.
```

Comment:
Where do we use this value? We shouldn't rely on this store on Stepper at all.

Reply:
I believe we don't need it at all, removing it.